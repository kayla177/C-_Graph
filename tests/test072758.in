ENTITY NFMAXALY name_NFMA conference
ENTITY 6F09X1TU name_6F09 institution
ENTITY FMR9T3YL name_FMR9 journal
ENTITY 5U4K0YAI name_5U4K author
ENTITY 1J7FAHJM name_1J7F institution
ENTITY OJXEQ5T8 name_OJXE paper
ENTITY AUJHEBEC name_AUJH institution
ENTITY SQB4GFHW name_SQB4 conference
ENTITY XHH0EVB1 name_XHH0 institution
ENTITY FIPQ20MC name_FIPQ author
RELATIONSHIP NFMAXALY presented_at XHH0EVB1 8.1
RELATIONSHIP 1J7FAHJM published_in FIPQ20MC 1.8
RELATIONSHIP FMR9T3YL authored_by 5U4K0YAI 4.7
RELATIONSHIP OJXEQ5T8 affiliated_with SQB4GFHW 0.3
RELATIONSHIP XHH0EVB1 affiliated_with SQB4GFHW 2.4
RELATIONSHIP 1J7FAHJM cited_by AUJHEBEC 7.5
RELATIONSHIP SQB4GFHW authored_by FIPQ20MC 3.1
RELATIONSHIP 1J7FAHJM cited_by 5U4K0YAI 4.4
RELATIONSHIP OJXEQ5T8 cited_by NFMAXALY 6.7
RELATIONSHIP XHH0EVB1 collaborated_with 5U4K0YAI 4.7
RELATIONSHIP NFMAXALY collaborated_with SQB4GFHW 3.4
RELATIONSHIP NFMAXALY published_in 6F09X1TU 2.7
RELATIONSHIP NFMAXALY published_in FMR9T3YL 6.7
RELATIONSHIP FIPQ20MC collaborated_with 6F09X1TU 5.0
RELATIONSHIP NFMAXALY affiliated_with AUJHEBEC 1.9
PATH FIPQ20MC 6F09X1TU
PRINT 5U4K0YAI
HIGHEST
PRINT 6F09X1TU
PATH NFMAXALY FMR9T3YL
PRINT FIPQ20MC
DELETE SQB4GFHW
DELETE AUJHEBEC
HIGHEST
PATH OJXEQ5T8 FMR9T3YL
EXIT
