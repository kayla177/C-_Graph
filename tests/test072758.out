success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
FIPQ20MC 6F09X1TU 5
FMR9T3YL 1J7FAHJM XHH0EVB1
1J7FAHJM OJXEQ5T8 23.9
NFMAXALY FIPQ20MC
NFMAXALY FMR9T3YL 6.7
1J7FAHJM SQB4GFHW 6F09X1TU
success
success
1J7FAHJM OJXEQ5T8 23.9
OJXEQ5T8 NFMAXALY FMR9T3YL 13.4
